tests/pr_test.go

```diff
@@ -75,6 +79,85 @@ func setupOptionsQuickStartPattern(t *testing.T, prefix string, dir string) *tes
 	return options
 }
 
+type RecursiveCopy struct {
```

Comment:
This struct doesn't technically need to be "public" by making it capitalized, it could be "private" by having it start with lower case.

Also the word "Copy" might be misleading here, as we are not actually copying any files.